frontend/pages/admin/pacta-version/[id].vue

```diff
@@ -37,6 +39,14 @@ const changes = computed<PactaVersionChanges>(() => {
   }
 })
 const hasChanges = computed<boolean>(() => Object.keys(changes.value).length > 0)
+const incompleteFields = computed<string[]>(() => editor.value?.incompleteFields ?? [])
+const isIncomplete = computed(() => incompleteFields.value.length > 0)
+const saveTooltip = computed<string | undefined>(() => {
+  if (!hasChanges.value) { return 'All changes saved' }
+  if (isIncomplete.value) { return `Cannot save with incomplete fields: ${incompleteFields.value.join(', ')}` }
+  return undefined
+})
+const saveDisabled = computed<boolean>(() => saveTooltip.value !== undefined)
```

Comment:
If the idea is that we copy and paste this to each page that has an editor on it, I'm not a fan, as this is a lot of boilerplate that we're going to want to update. I think a system roughly like the following should work and is much clearer + cleaner:

On the page:
```vue
<script setup lang="ts">
// ...
const { newForm } = useForm() // A composable that doesn't currently exist.
const form = newForm('maybeSomeStateLikeKeyThing')
// ...
</script>

<template>
<!-- ... -->

<PactaversionEditor
  :form="form"
  v-model=pactaVersion="pactaVersion"
/>

<!-- ... -->
<div v-tooltip.bottom="form.saveTooltip">
<PVButton
  :disabled="form.saveDisabled" // or name it whatever
  ...
/>
</div>
</template>
```

In the editor component:

```vue
<script setup lang="ts">
const props = defineProps<{
  pactaVersion: PactaVersion
  form: TheFormType
}>()

// Can also specify .length > 0 here, or just let truthy/falsy be decider of validity
props.form.register('Version Name', () => model.value.name)
props.form.register('Docker Image Digest', () => model.value.digest)
</script>
```

Does that make sense? Benefits I see:

1. Most boilerplate now lives in one place in a composable. Makes updating, tweaking, adding functionality easier
2. No need to expose functions on child component, which always felt like a last resort to me.

The only downside I can think of is that the devil is in the details implementation-wise, but I still think it should be pretty straightforward.